### ISSUE TITLE
Missing Morpheus Icons

### ISSUE THREAD

**Issue by b4D8** [YunoHost-Apps/matomo_ynh#20]
**Describe the bug**
[Morpheus icons](https://github.com/matomo-org/matomo-icons) are missing (or at least not found - NGINX returns a 404 error when I try to directly open an image for instance : /plugins/Morpheus/icons/dist/browsers/UNK.png)

This might be related to this issue: https://issues.matomo.org/index.php/13030 regarding dependencies in git submudules.

**Versions**
- Hardware: VPS OVH
- YunoHost version: stable 3.0.8.3
- I have access to my server: Through SSH, through the webadmin, direct access via keyboard / screen
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance ?: No
- Using: stable 3.14.1

**Expected behavior**
Seeing icons in respective placeholders

![missing morpheus icons](https://user-images.githubusercontent.com/56940406/101331183-ea67d580-3873-11eb-80c0-9c6022f3b6c9.png)

**Comment by ericgaspar** [comment]
Is the problem of missing icons still present in testing version 4.0.5?
```
sudo yunohost app install https://github.com/YunoHost-Apps/matomo_ynh/tree/testing
or
sudo yunohost app upgrade matomo -u https://github.com/YunoHost-Apps/matomo_ynh/tree/testing 
```

**Comment by b4D8** [comment]
Yes, I can confirm the problem is still present in 4.0.5... plus a new one, more problematic, that I referenced in a new issue #25.

**Comment by ericgaspar** [comment]
Morpheus icons missing problem should be resolve on testing branche.
```
sudo yunohost app install https://github.com/YunoHost-Apps/matomo_ynh/tree/testing
or
sudo yunohost app upgrade matomo -u https://github.com/YunoHost-Apps/matomo_ynh/tree/testing
```
As always, if anyone can try and report errors, it will be greatly appreciated.

**Comment by ericgaspar** [comment]
Should be fixed with #24
